client/wildcard/src/components/Typography/Heading/Heading.tsx

```diff
@@ -14,11 +14,11 @@ type InternalHeadingProps = HeadingProps & {
     styleAs?: HeadingElement
 }
 
-const getStyleAs = (headerX: HeadingElement | undefined): string | undefined =>
+const getStyleAs = (headerX: React.ElementType | undefined): string | undefined =>
```

Comment:
```suggestion
const getStyleAs = (headerX: HeadingElement | undefined): string | undefined =>
```

Can we fix this file so we no longer support using `div` as a heading? I feel like we'd never want to encourage that. We could just default `Component = 'h1'` here instead (and update the ForwardRef type)